tests/e2e/test_launch.py

```
@@ -0,0 +1,111 @@
+            text=True,
+            timeout=10,
+        )
+
+        if result.returncode != 0:
+            assert False, f"Package import failed:\n{result.stderr}"
+
+        print("✅ Package imports successfully")
+
+    def test_streamlit_app_starts_without_errors(self):
+        """Test that Streamlit starts and serves content without errors."""
+        process = subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)